scripts/ci/config.yml

```diff
@@ -206,6 +210,8 @@ transformers:
     - "tensorflow==2.5.0"
     - "importlib_metadata"
     - "Pillow==8.4.0"
+  external_scripts: |
+    python -m pip install "typing_extensions>=3.10"
```

Comment:
can we just put `"typing_extensions>=3.10"` in the dependencies list above?

Reply:
it would resolve in pip conflict